vulture_os/toolkit/api_parser/harfanglab/harfanglab.py

```diff
@@ -233,7 +255,9 @@ def execute(self):
                 # Writting may take some while, so refresh token in Redis
                 self.update_lock()
 
-        self.frontend.last_api_call = to
-        self.frontend.save()
+            if not is_threshold_trigerred:
+                self.frontend.harfanglab_timestamps[f"since_{log_type}"] = self.frontend.harfanglab_timestamps[f"to_{log_type}"]
+                self.frontend.harfanglab_timestamps[f"to_{log_type}"] = min(timezone.now() - timedelta(minutes=2), self.frontend.harfanglab_timestamps[f"to_{log_type}"] + timedelta(hours=24))
+                self.frontend.save()
```

Comment:
Still need to update the 'last_api_call' value, even if it's not directly used (for human analysis and monitoring)